### PR TITLE
Tweak editor menu button styling

### PR DESCRIPTION
**Pull request by tburry**
Related vanilla/rich-editor#88

### CODE REVIEW

File: applications/dashboard/design/style-compat.css [hunk 1]
```diff
@@ -832,9 +830,9 @@ label.checkbox input.checkbox-input {
 }
 
 .richEditor-menuItems .richEditor-menuItem .richEditor-button, .richEditor-menuItems .richEditor-menuItem.richEditor-button {
-  width: 39px;
-  font-size: 24px;
-  line-height: 39px;
+  width: 36px;
+  font-size: 22.1538461538px;
```
Comment (charrondev) [inline applications/dashboard/design/style-compat.css]: Where is this value coming from? I don't see it in the changed source.

File: applications/dashboard/scss/editor/_richEditor.scss [hunk 2]
```diff
@@ -77,7 +77,7 @@
     flex-wrap: nowrap;
     list-style: none;
     padding: 0;
-    width: calc(100% - #{2 * $richEditor-border_width});
+    //width: calc(100% - #{2 * $richEditor-border_width});
```
Comment (charrondev) [inline applications/dashboard/scss/editor/_richEditor.scss]: What's up with this commented line?

File: applications/dashboard/design/style-compat.css (hunk 1 above)
Comment (tburry) [reply applications/dashboard/design/style-compat.css]: Sorry, this was meant to be given over to @slafleche to complete. Not ready for a review.

File: applications/dashboard/scss/editor/_richEditor.scss (hunk 2 above)
Comment (tburry) [reply applications/dashboard/scss/editor/_richEditor.scss]: I'm guessing that's also the reason for the whacky font size, but it does cause the menu bars to be incorrect widths so things need to change there.

**Comment by charrondev** [comment]
Supplanted by https://github.com/vanilla/vanilla/pull/7634

**Pull request closed without merging**